src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -6,6 +6,16 @@
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null || user.getId() == null
+                || user.getLogin() == null || user.getAge() == null
+                || user.getPassword() == null || user.getAge() < 18
+                || user.getAge() <= 0 || user.getLogin().length() < 6
```

Comment:
do we need this check (user.getAge() <= 0) if we have this one user.getAge() < 18 ?